periphery/src/managers/UniswapNFTManager.sol

```diff
@@ -65,6 +65,9 @@ contract UniswapNFTManager is IManager {
                     deadline: block.timestamp
                 })
             );
+
+            token0.safeTransfer(owner, collected0 - burned0);
+            token1.safeTransfer(owner, collected1 - burned1);
```

Comment:
But now we need to send leftover funds back to the owner -- the burned amounts are used to re-create the Uniswap position, but fees (collected - burned) need to go somewhere. Previously they would've just sat in the `Borrower`, now we're sending them to the owner.